pkg/backup/item_backupper_test.go

```diff
@@ -123,6 +123,35 @@ func TestBackupItemSkipsClusterScopedResourceWhenIncludeClusterResourcesFalse(t
 	assert.NoError(t, err)
 }
 
+func TestBackupItemWhenInTerminatingStatus(t *testing.T) {
+	f := true
+	ib := &defaultItemBackupper{
+		backup: &v1.Backup{
+			Spec: v1.BackupSpec{
+				IncludeClusterResources: &f,
+			},
+		},
+		namespaces: collections.NewIncludesExcludes(),
+		resources:  collections.NewIncludesExcludes(),
+	}
+
+	u := unstructuredOrDie(`{"apiVersion":"v1","kind":"Foo","metadata":{"name":"bar", "namespace": "test", "deletionTimestamp": "timestamp"},  "status": {
```

Comment:
Please use a real `deletionTimestamp`. You can probably remove the `conditions` too.

Reply:
Done.